.github/workflows/dart-desktop-tests.yml

```diff
@@ -77,7 +77,9 @@ jobs:
           clusterName: ${{ env.BAAS_CLUSTER }}
 
       - name: Run tests
-        run: ${{ inputs.architecture == 'arm' && 'arch -arm64 ' || '' }}dart test -r expanded --coverage ./coverage/ -j 1 --test-randomize-ordering-seed random
+        run: |
+          ${{ inputs.architecture == 'arm' && 'arch -arm64 ' || '' }}dart test -x destructive -r expanded --coverage ./coverage/ -j 1 --test-randomize-ordering-seed random
+          ${{ inputs.architecture == 'arm' && 'arch -arm64 ' || '' }}dart test  test/destructive_schema_test.dart -r expanded
```

Comment:
Why do we invoke it two times. Isn't the above enough?

Reply:
The first command excludes tests with tag 'destructive'. After all the other tests pass then we run another command for testing only the destructive tests. Otherwise, 'destructive' schema change tests terminates the flexible sync and this breaks the rest of the tests. I will put this as a comment in the yml file.

Reply:
I run the 'destructive' tests only for Dart. I think it is enough.

Reply:
My question is aren't the test with `destructive` tag the same as the tests in `test/destructive_schema_test.dart`?

Reply:
ok this actually excluded in the first run. resolving